langchain/examples/price_deal_finder/price_deal_finder.py

```diff
@@ -0,0 +1,50 @@
+import asyncio
+import getpass
+import os
+
+from langchain.chat_models import init_chat_model
+from langchain_agentql import AgentQLBrowserToolkit
+from langchain_agentql.utils import create_async_playwright_browser
+from langchain_community.tools.playwright import ClickTool, NavigateTool
+from langgraph.prebuilt import create_react_agent
+
+
+async def main():
+
+    # Instantiate LLM
+    if not os.environ.get("OPENAI_API_KEY"):
+        os.environ["OPENAI_API_KEY"] = getpass.getpass("Enter API key for OpenAI: ")
```

Comment:
we usually don't use getpass, and rely on env vars, in every AQL example, for example. Let's keep it consistent